### PR TITLE
Add a knmstate lane for future nmstate version

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml [hunk 1]
```diff
@@ -63,6 +63,39 @@ presubmits:
...
+            resources:
+              requests:
+                memory: "29Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "NMSTATE_PIN=future automation/check-patch.e2e-k8s.sh"
+
+
```
Comment (rhrazdil) [inline github/ci/prow/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml:98]: nit: extra newline

**Review by rhrazdil — CHANGES_REQUESTED** [review]

**Comment by phoracek** [comment]
/overried  rehearsal-pull-kubernetes-nmstate-e2e-handler-k8s-future

**Comment by phoracek** [comment]
/override rehearsal-pull-kubernetes-nmstate-e2e-handler-k8s-future

**Pull request merged**